01-js-intro-and-jquery/app.js

```diff
@@ -0,0 +1,27 @@
+var main = function() {
+    
+    $('.hidden').fadeIn(5000, function() {
+
+        $( '.alias' ).focus()
+    });
+
+    $('#btn').click( function( event ) {
+
+        $.get("http://bootcamp.aws.af.cm/welcome/yourname", function(text) {
```

Comment:
Have in mind using ``$.ajax`` with its callbacks or Promises (this will allow you to solve exercise 9 ;)). We'll talk this in detail in a call, don't worry right now, it's just in case you want to check documentation. :+1: